eventing/triggers/create-triggers.adoc

```diff
@@ -1,11 +1,65 @@
 :_content-type: ASSEMBLY
-include::_attributes/common-attributes.adoc[]
 [id="create-triggers"]
 = Creating triggers
+include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-include::snippets/serverless-brokers-intro.adoc[]
+Triggers in Knative Eventing allow you to route events from a broker to a specific subscriber based on your requirements. By defining a Trigger, you can connect event producers to consumers dynamically, ensuring events are delivered to the correct destination. 
+
+This section describes the steps to create a Trigger, configure its filters, and verify its functionality. Whether you're working with simple routing needs or complex event-driven workflows. 
+
+The following examples displays common configurations for Triggers, demonstrating how to route events to Knative services or custom endpoints.
+
+.Example of routing events to a Knative Serving service
+
+The following Trigger routes all events from the default broker to the Knative Serving service named `my-service`:
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: my-service-trigger
+spec:
+  broker: default
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: my-service
+----
```

Comment:
Could we add a note here, that routing all events - without a filter - is a good practise for debugging?

and than perhaps link to the "advanced-trigger-filters" document? For like "to learn more about filtering, click here" ?

Reply:
Can you please provide me with an example of this?

Reply:
I have included a basic filter in the example. Can you please verify now?